db/db_impl/db_impl.h

```diff
@@ -1005,6 +1005,10 @@ class DBImpl : public DB {
   // is only for the special test of CancelledCompactions
   Status WaitForCompact(bool waitUnscheduled = false);
 
+  void NewDescriptorLogForNextManifestWrite() {
```

Comment:
Let's call this `NewManifestOnNextUpdate()`. 'Descriptor log' is an old name for manifest file.